src/string/formatter.cr

```diff
@@ -110,7 +110,7 @@ struct String::Formatter(A)
 
   private def consume_width(flags)
     case current_char
-    when '1'..'9'
+    when '0'..'9'
```

Comment:
I think this is wrong, unless the 0 is after a non-0 digit, 0 is a separate flag: https://github.com/crystal-lang/crystal/blob/391785249f66aaf5929b787b29810aeb4af0e1e8/src/kernel.cr#L150

(_I can't fully check the implementation, i'm on my phone_)